slices/procps.yaml

```diff
@@ -0,0 +1,49 @@
+package: procps
+
+essential:
+  - procps_copyright
+
+slices:
+  bins:
+    essential:
+      - init-system-helpers_bins
```

Comment:
It seems to me that the reason `init-system-helpers` is a dependency of the `procps` package is due to the usage of `update-rc.d` script in the `postinst` script. Since we are not doing that in our mutation scripts, I do not think the `init-system-helpers_bins` is necessary here.

I see https://github.com/canonical/chisel-releases/pull/234/files#r1602980884, but I don't think it's necessary. If some application requires it, then they can always install the `init-system-helpers_bins` slice directly; no need to make the `procps_bins` slice fat.